deploy/vagrant/create

```diff
@@ -147,7 +154,7 @@ if [[ "$ALIAS_IP" == "true" ]]; then
   TMP_FILE=./alluxio_hosts_tmp_file
   if [[ -f $ANSIBLE_INFO_FILE ]]; then
     echo -e "\n# Generated by Alluxio vagrant deploy script" > $TMP_FILE
-    if [[ "$PROVIDER" == "aws" ]]; then
+    if [[ ( "$PROVIDER" == "aws" ) || ( "$PROVIDER" == "google" ) ]]; then
```

Comment:
My only question at this point is how to address this. Should I deviate with how we've done aws? Are you suggesting it should be:
if [[ ( "$PROVIDER" == "aws" ) || ( "${PROVIDER}" == "google" ) ]]; then